lib/text/bidi_text.flow

```diff
@@ -26,19 +26,80 @@ export {
 
 	// Returns a reversed list with elements of original list wrapped into a pair with text flow direction
 	markListToBiDiReverse(list: List<?>, getText: (?) -> string) -> List<Pair<?, int>>;
+	CharCatSplitPosText(
```

Comment:
Add comments to this structure to explain what each field is

Reply:
done